kbs/tools/client/Cargo.toml

```diff
@@ -18,13 +18,20 @@ base64.workspace = true
 clap = { version = "4.0.29", features = ["derive"] }
 env_logger.workspace = true
 jwt-simple = "0.11.4"
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev = "42b7c96", default-features = false }
+kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev = "fc89b71", default-features = false }
 log.workspace = true
-reqwest = { version = "0.11.18", default-features = false, features = ["cookies", "json"] }
+reqwest = { version = "0.11.18", default-features = false, features = [
```

Comment:
nit: we should avoid formatting the Cargo.toml file. edit w/ nano or something :)